packages/create-relay-app/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "create-relay-app",
+  "description": "Easy configuration of Relay for existing projects",
+  "version": "0.0.0",
+  "keywords": [
+    "create-relay-app",
+    "graphql",
+    "relay",
+    "react",
+    "vite.js",
+    "next.js",
+    "create react app"
+  ],
+  "license": "MIT",
+  "homepage": "https://relay.dev",
+  "bugs": "https://github.com/facebook/relay/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/relay.git",
+    "directory": "packages/create-relay-app"
+  },
```

Comment:
I think we want to rename it to "install-relay".

Reply:
If we rename it to something that doesn't start with `create-`, we will loose the ability for it to be run as `npm/yarn/pnpm create relay-app`. If it's called `install-relay`, users will have to call it with `npx install-relay` and `pnpm/yarn dlx install-relay`. Users of yarn v1 will have no way of executing it and will have to fallback on npx.
But I reckon `create-relay-app` isn't really accurate, since we are not creating, but enhancing an existing application. I mainly chose `create-`, because it feels "official" and users are familiar with it.